apps/src/code-studio/components/ModelManagerDialog.jsx

```diff
@@ -0,0 +1,76 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import Button from '@cdo/apps/templates/Button';
+import Applab from '@cdo/apps/applab/applab';
```

Comment:
I'm not sure if this is the cause of your storybook problems, but it seems like the code you are adding should not be relying on applab like this. Looking at the chain of dependencies, this is going to make it so that every level type which uses CodeWorkspace is now going to be importing all of App Lab. One idea would be to break this dependency, and then see if you still have the same storybook problems. Please let me know if you need more info on how to accomplish this.

Reply:
I guess I'll offer some info anyway :-) One way would be to "lazy load" the ModelManagerDialog, like this: https://github.com/code-dot-org/code-dot-org/blob/5bd80f434e6d5cb8fd3e24eb4de42d7c3acfd4ff/apps/src/code-studio/assets/show.js#L7

there may also be a simpler way to cut out the dependencies, but it might take a bit more understanding of your code than I currently have to figure out where.

Reply:
aha! yes, this import of Applab does seem to be the culprit, though I still don't understand the full intricacies of how.  Thank you! I'll give lazy loading a try.